make/modules/java.base/gensrc/GensrcCharsetMapping.gmk

```diff
@@ -52,5 +76,10 @@ $(CHARSET_DONE_BASE)-stdcs: $(CHARSET_DATA_DIR)/charsets \
 	    $(CHARSET_COPYRIGHT_HEADER) \
             $(LOG_DEBUG)
 	$(TOUCH) '$@'
+	$(ECHO) OpenJ9Logs0725-1428pm - '$@'
+	$(MKDIR) -p $(SUPPORT_OUTPUTDIR)/overlay-gensrc/src/java.base/sun/nio/cs
+	$(CP) $(CHARSET_GENSRC_JAVA_DIR_BASE)/StandardCharsets.java $(SUPPORT_OUTPUTDIR)/overlay-gensrc/src/java.base/sun/nio/cs
+	$(call RunJPP, JAVA$(VERSION_FEATURE), $(SUPPORT_OUTPUTDIR)/overlay-gensrc, /overlay-result)
+	$(CP) $(SUPPORT_OUTPUTDIR)/j9jcl/overlay-result/java.base/sun/nio/cs/StandardCharsets.java $(CHARSET_GENSRC_JAVA_DIR_BASE)
```

Comment:
This should go before the `$(TOUCH)` command above so failures won't be ignored on retry.

Reply:
JPP output should go somewhere other than `$(J9JCL_SOURCES_DIR)/overlay-result` lest it be confused for a module.

Reply:
Changed the output folder to `$(SUPPORT_OUTPUTDIR)/overlay-result`.